redbot/cogs/audio/audio.py

```diff
@@ -2671,6 +2746,33 @@ def _track_creator(player, position=None, other_track=None):
             track_obj[key] = value
         return track_obj
 
+    @staticmethod
+    def _track_limit(ctx, track, maxlength):
+        try:
+            length = round(track.length / 1000)
+        except AttributeError:
+            length = round(track / 1000)
+        if length > 259200:
```

Comment:
I'm curious about this and if it might have potential problems. Is this in case we're playing a live stream we don't care about the maxlength? Will this cause issues for streams that don't return a value of 72 hours?

Reply:
Decided to move this value more towards the upper limit instead and added a comment regarding livestream length.